src/decorator/live-decorator.ts

```diff
@@ -10,19 +10,19 @@ const logger = getContribLogger({
 
 export class KalturaLiveEngineDecorator 
   implements KalturaPlayerTypes.IEngineDecorator {
-  _plugin: KalturaLivePlugin;
-  _engine: any;
-  _dispatcher: Function;
-  _hadError: boolean;
+  _plugin: KalturaLivePlugin | null = null;
```

Comment:
all this initialization is not needed, we have tried something and forgot to remove it